server/prisma/migrations/20240210185024_init/migration.sql

```diff
@@ -1,3 +1,292 @@
+    "newsletter" BOOLEAN DEFAULT false,
+    "createddate" TIMESTAMPTZ(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "contacts_pkey" PRIMARY KEY ("contactid")
+);
+
+-- CreateTable
+CREATE TABLE "date" (
+    "dateid" INTEGER NOT NULL,
+    "date_actual" DATE NOT NULL,
+    "day_name" VARCHAR(9) NOT NULL,
+    "day_of_week" INTEGER NOT NULL,
+    "day_of_month" INTEGER NOT NULL,
+    "day_of_quarter" INTEGER NOT NULL,
+    "day_of_year" INTEGER NOT NULL,
+    "week_of_month" INTEGER NOT NULL,
+    "week_of_year" INTEGER NOT NULL,
+    "month_actual" INTEGER NOT NULL,
+    "month_name" VARCHAR(9) NOT NULL,
+    "quarter" INTEGER NOT NULL,
+    "year_actual" INTEGER NOT NULL,
+    "first_day_of_week" DATE NOT NULL,
+    "last_day_of_week" DATE NOT NULL,
+    "first_day_of_month" DATE NOT NULL,
+    "last_day_of_month" DATE NOT NULL,
+    "first_day_of_quarter" DATE NOT NULL,
+    "last_day_of_quarter" DATE NOT NULL,
+    "first_day_of_year" DATE NOT NULL,
+    "last_day_of_year" DATE NOT NULL,
+    "weekend" BOOLEAN NOT NULL,
+
+    CONSTRAINT "date_pkey" PRIMARY KEY ("dateid")
+);
+
+-- CreateTable
+CREATE TABLE "discounts" (
+    "discountid" SERIAL NOT NULL,
+    "code" VARCHAR(32) NOT NULL,
+    "active" BOOLEAN NOT NULL,
+    "amount" MONEY,
+    "percent" INTEGER,
+    "tickettypeid_fk" INTEGER,
+    "usagelimit" INTEGER,
+    "min_tickets" INTEGER,
+    "min_events" INTEGER,
+
+    CONSTRAINT "discounts_pkey" PRIMARY KEY ("discountid")
+);
+
+-- CreateTable
+CREATE TABLE "donations" (
+    "donationid" SERIAL NOT NULL,
+    "orderid_fk" INTEGER NOT NULL,
+    "anonymous" BOOLEAN NOT NULL DEFAULT false,
+    "amount" MONEY NOT NULL,
+    "frequency" "freq" NOT NULL,
+    "comments" VARCHAR(255),
+
+    CONSTRAINT "donations_pkey" PRIMARY KEY ("donationid")
+);
+
+-- CreateTable
+CREATE TABLE "eventinstances" (
+    "eventinstanceid" SERIAL NOT NULL,
+    "eventid_fk" INTEGER NOT NULL,
+    "eventdate" INTEGER NOT NULL,
+    "eventtime" TIMETZ(6) NOT NULL,
+    "salestatus" BOOLEAN NOT NULL DEFAULT true,
+    "totalseats" INTEGER NOT NULL DEFAULT 0,
+    "availableseats" INTEGER NOT NULL DEFAULT 0,
+    "detail" VARCHAR(255),
+    "purchaseuri" VARCHAR(255),
+    "ispreview" BOOLEAN NOT NULL DEFAULT false,
+    "deletedat" TIMESTAMP(3),
+
+    CONSTRAINT "eventinstances_pkey" PRIMARY KEY ("eventinstanceid")
+);
+
+-- CreateTable
+CREATE TABLE "events" (
+    "eventid" SERIAL NOT NULL,
+    "seasonid_fk" INTEGER,
+    "eventname" VARCHAR(255) NOT NULL,
+    "eventdescription" VARCHAR(255) NOT NULL,
+    "active" BOOLEAN NOT NULL DEFAULT false,
+    "seasonticketeligible" BOOLEAN NOT NULL DEFAULT true,
+    "imageurl" VARCHAR(255),
+    "deletedat" TIMESTAMP(3),
+
+    CONSTRAINT "events_pkey" PRIMARY KEY ("eventid")
+);
+
+-- CreateTable
+CREATE TABLE "order_ticketitems" (
+    "id" SERIAL NOT NULL,
+    "orderid_fk" INTEGER NOT NULL,
+    "price" MONEY NOT NULL,
+
+    CONSTRAINT "order_ticketitems_pkey" PRIMARY KEY ("id")
+);
```

Comment:
This table appears to just be an intermediary between an order and a ticketitem, so could we skip it altogether and record the price with the ticketitem instead? And side note for a future PR, the idea of another table for recording ticket exchanges makes sense to me where we include a fk to it in the ticketitem as well

Reply:
Currently I have both `orderticketitems` and `subscriptionticketitems` (will be added in another PR) "derived" from `ticketitems` because they share the same base data but differ in what they are related to., but that can be changed.